config/test.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
   allowErrorSimulation: true,
   enableClientConsole: true,
   isDeployed: false,
+  enableLogging: false,
```

Comment:
I don't think this is helpful because you can already run the tests with `yarn test --silent`.

I would rather [patch up Jest](https://github.com/facebook/jest/issues/4156) to expose its DefaultReporter so we can write a reporter that only shows logging for failing tests.

Reply:
oh really? ok, let's remove it then.